### PR TITLE
Updated translation for 0.13

### PR DESCRIPTION
**Pull request by rpinola**

### CODE REVIEW

**Comment by erciccione** [comment]
hey @rpinola looks like you have some conflicts here, you need to use an updated clone of this repo before adding the changes.

**Comment by rpinola** [comment]
> hey @rpinola looks like you have some conflicts here, you need to use an updated clone of this repo before adding the changes.

How do you do that, in ELI5 format?

**Comment by erciccione** [comment]
All these translations are obsolete and the files you worked on are not compatible with the present one. Btw, i still have push access to your repo from last time, i could reset your master branch to make it even with the official one, but i'm afraid you will have to start from the beginning (only about 50 strings are missing btw)

**Comment by rpinola** [comment]
> All these translations are obsolete and the files you worked on are not compatible with the present one. Btw, i still have push access to your repo from last time, i could reset your master branch to make it even with the official one, but i'm afraid you will have to start from the beginning (only about 50 strings are missing btw)

Ok, I'll do it again. Just give the instructions how to fork the updated version?

**Comment by erciccione** [comment]
@rpinola did you close this by mistake or on purpose? i see the commit
